Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript

```diff
@@ -68,6 +68,112 @@ local sTableObjectsA
 local sScrollerId
 local sPollID
 
+--------------------------------------------------------------------------------
+-- DataGrid 2 Variables
+
+local sEditMode
+
+---
+
+local sReorderInProgress
+local sReorderControl
+local sReorderStartIndex
+local sReorderStartLine
+local sReorderLastHoverLine
+local sReoderMouseMoveInProgress
+
+constant kReorderDragControlBlendLevel = 30
+constant kReorderDragControlGlowColor = "0,0,0"
+constant kReorderDragControlGlowSize = 10
+constant kReorderDragControlGlowSpread = 10
+
+---
+
+local sReorderScrollPollMsgID
+
+constant kReorderScrollPollRate = 30
+constant kReoderScrollPollMargin = 30
+constant kReorderScollPollIncrement = 10
+
+---
+
+local sAnimationsA
+local sAnimationsLastID
+local sAnimationsPulseMsgID
+
+constant kAnimationsPulseRate = 30
+---
+
+local sAnimationProps
+
+constant kEditModeEnterExitDuration = 250
+constant kEditModeEnterExitActionSelectControlControlEasing = "in out"
+constant kEditModeEnterExitReorderControlEasing = "linear"
+
+constant kEditModeActionControlAnimationDuration = 250
+constant kEditModeActionControlAnimationEasing = "in out"
+
+constant kReorderAnimationDuration = 250
+constant kReorderAnimationEasing = "in out"
+
+constant kReorderHomingDuration = 250
+constant kReorderHomingEasing = "in out"
+
+constant kDeleteIndexAnimationDuration = 250
+constant kDeleteIndexAnimationEasing = "in out"
+
+constant kSwipeCompleteDuration = 300
+constant kSwipeCompleteEasing = "out back"
+
+constant kSwipeReturnDuration = 500
+constant kSwipeReturnEasing = "out bounce"
+
+---
+
+local sCustomisableControlsInitialised
+local sCustomisableControlsA
+local sConstomisableControlsSingletonsA
+local sConstomisableControlsSingletonCopyMapA
```

Comment:
Some `constomisable` rather than `customisable`s here

Reply:
It's not even close. Where did the n come from?

Reply:
Haha, looks like your fingers got so used to typing `constant`. I do it all the type with `local` when I'm trying to lock messages- `locak messages` *all* the time